lib/getstats.js

```diff
@@ -582,6 +606,87 @@ function standardizeFirefoxStats(response, isRemote) {
   return standardizedStats;
 }
 
+/**
+ * Standardize the MediaStreamTrack's statistics in Safari.
+ * @param {RTCStatsReport} response
+ * @param {MediaStreamTrack} Track
+ * @returns {StandardizedTrackStatsReport}
+ */
+function standardizeSafariStats(response, track) {
+  var trackStats = Array.from(response.values()).find(function(stat) {
+    return stat.type === 'track' && stat.trackIdentifier === track.id;
+  });
+
+  // NOTE(syerrapragada): trackIdentifier in RTCStatsReport
+  // for remote tracks doesn't match the MediaStreamTrack's ID.
+  // see: https://bugs.webkit.org/show_bug.cgi?id=174519
+  if (!trackStats) {
```

Comment:
@syerrapragada So are we going to always report empty RemoteTrack stats?

NOTE: This is apparently fixed for unified plan.

Reply:
Yes